app/javascript/components/teams-dropdown.vue

```diff
@@ -12,19 +12,20 @@
 <script>
 import ClickableDropdown from './clickable-dropdown';
 import { PROCESS_NEW_TEAM } from '../store/action-types';
+import { mapState } from 'vuex';
```

Comment:
'mapState' is defined but never used                  no-unused-vars<br>Absolute imports should come before relative imports  import/imports-first